config/crd/kustomization.yaml

```
@@ -5,22 +5,10 @@ resources:
 - bases/controller.devfile.io_components.yaml
 - bases/controller.devfile.io_workspaceroutings.yaml
 - bases/workspace.devfile.io_devworkspaces.yaml
+- bases/workspace.devfile.io_devworkspacetemplates.yaml
 # +kubebuilder:scaffold:crdkustomizeresource
 
 patchesStrategicMerge:
```

IDE suggests that it should be array
```suggestion
patchesStrategicMerge: []
```

---

Removed the whole thing, thanks